vela-templates/addons/fluxcd.yaml

```diff
@@ -1,2158 +0,0 @@
-apiVersion: core.oam.dev/v1beta1
-kind: Initializer
```

Comment:
please keep a initializer file so others can apply instead of using vela cli

Reply:
We should do it automatically

Reply:
After communicating with @chivalryq  offline, we will use script to automatically generate this file.

Reply:
done